### ISSUE TITLE
Bar not pushed when MeshSettings1D fragment added without a Bar Release

### ISSUE THREAD

**Issue by michaellisitsa** [BHoM/Lusas_Toolkit#311]
**Description of problem**

With BHoM v4.0, "Modify.AddFragment" allows you to add MeshSettings1D to a bar being pushed, however adding this fragment by the method shown below causes the Bar push to fail if the user has not inputed "Releases".

This error occurs when the Bar Object is from [BHoM/Bar.cs](https://github.com/BHoM/BHoM/blob/master/Structure_oM/Elements/Bar.cs) because it doesn't have a default argument, even though the description on the node says it does. The [Engine/Bar.cs](https://github.com/BHoM/BHoM_Engine/blob/master/Structure_Engine/Create/Elements/Bar.cs) on the other hand has a conditional operator to catch a null value.

Details of the warning above the "Push" tab.
``` C#
Warning: Compute.RunCaller operation failed. 
This component failed to run properly.
- Error: Object reference not set to an instance of an object.
- Occured in  BH.Adapter.Lusas.LusasV19Adapter.<>c.<CreateCollection>b__74_1(Bar m)
     called from BH.Adapter.Lusas.LusasV19Adapter.CreateCollection(IEnumerable`1 bars)
- Are you sure you have the correct type of inputs? Check their description for more details.
```

**How to reproduce**
![image](https://user-images.githubusercontent.com/58811856/104090435-db3cd300-52ca-11eb-830c-9595f4b64bd1.png)
![image](https://user-images.githubusercontent.com/58811856/104090441-ec85df80-52ca-11eb-8193-2d087670d963.png) 

**Solution**

Add a default argument for the BHoM command.

**Setup**
BHoM 4.0
Dynamo 2.0.4
Revit 2019
LUSAS 19.0.3

**Comment by peterjamesnugent** [comment]
Hi @michaellisitsa, thanks for raising this. Would you be interested in raising a PR to squash this bug and I can review?

**Comment by peterjamesnugent** [comment]
@michaellisitsa I have tested this on the master branch and I cannot reproduce the error you got. I believe this was closed out in #312 when a series of `null` checks were added to the `Create` methods. 

Therefore, I am going to close this issue for now, feel free to reopen it if you run if the issue persists.
